curriculum/challenges/english/03-front-end-libraries/bootstrap/label-bootstrap-buttons.english.md

```diff
@@ -7,7 +7,8 @@ challengeType: 0
 ## Description
 <section id='description'>
 Just like we labeled our wells, we want to label our buttons.
-Give each of your <code>button</code> elements text that corresponds to its <code>id</code>'s selector.
+Give each of your <code>button</code> elements text that corresponds to its <code>id</code>'s selector prefixed with the "#" character.
+That is, for the <code>button</code> with id <code>target1</code> the label should be <code>#target1</label>.
```

Comment:
This line seems to be giving to much of the answer.  Something like this would be better place in the Guide article for the challenge as a hint.

Reply:
Looking at it now I do agree with you. However when I posted it I was also learning the subject and the question seemed too vague for me. I'm working on a guide topic and I'll push it to this PR.

Reply:
@jefferson2z Are you ok with closing this PR?

Reply:
Yes I agree with closing it.